python/ccxt/base/decimal_to_precision.py

```diff
@@ -18,19 +18,19 @@
 
 
 # rounding mode
-TRUNCATE = 0
-ROUND = 1
+ROUND = 0
```

Comment:
@samgermain Yeah this is an inconsistency we have had for a long time now, the only concern here is to break code in the userland if they're relying on these values. cc @ttodua

Reply:
@carlosmiei I don't know what to do. I know that whatever bug we have in lib, we should fix it. however, this is caveat as it might break userland codes. 
that's why I was strongly suggesting https://github.com/ccxt/ccxt/pull/17040 to avoid such manual comparisons (as 99% of users wouldnt import and directly compare to numeric values, like `if exchange.precisionMode == 1` or alike, while we could have safe approach `if exchange.precisionMode == exchange.TICK_SIZE` or etc. )

Reply:
I am not sure I understand what harm it would have caused if those 3 props (TICK_SIZE, DECIMAL_PLACES and SIGNIFICANT_DIGITS) would lived in exchange bases

Reply:
or add i.e. `isPrecisionTickSize: bool, isPrecisionDecimalPlaces: bool ...` props to exchange instances, so it would have been slightly easier.